Fixed hero section, navbar and landing page sections UI/UX (Fixes #3)

Sandraa16012007 opened the pull request:
###  Fix: Redesign & Improve Hero Section UI/UX  

#### 🐛 Related Issue  
**Closes #3**

---

#### Description  
This PR addresses the **Hero Section UI/UX issues** by completely redesigning the layout and improving visual appeal, responsiveness, and user navigation. The update introduces modern design elements, clear CTAs, and cohesive styling across the hero, navbar, and footer sections.  

---

#### Changes Made  
- **Navbar Improvements**  
  - Fixed spacing, alignment, and added hover interactions.  
  - Implemented sticky behavior for better navigation across pages.  
  - Unified design with consistent typography and colors.  

- **Hero Section Redesign**  
  - Added engaging **Call-to-Action (CTA)** buttons such as “Explore Programs” and “Get started”.  
  - Introduced a **modern layout** with visuals and improved text hierarchy.  
  - Enhanced typography for contrast and readability.  
  - Implemented **responsive design** to ensure proper scaling across all devices.  
  - Improved color scheme and background to make it visually appealing and dynamic.  

- **Supporting Content Enhancements**  
  - Added tagline, highlights, and a value proposition to make the section more informative and inviting.  
  - Optimized spacing and layout for a clean, balanced look.  

- **Footer & Other Section Updates**  
  - Redesigned the footer to align with the new theme and structure.  
  - Refactored styles for better consistency and alignment throughout the website.  

---

####  How to Test  
1. Go to the home/landing page.  
2. Observe the **hero section** — check that CTAs, text, and visuals are properly aligned and appealing.  
3. Scroll through the page — confirm that the **navbar remains sticky** and responsive.  
4. Resize the window or open on different devices — verify the layout adjusts correctly on all screen sizes.  
5. Inspect the **footer and other sections** for consistent styling and alignment.  

---

#### Preview (Optional)  
<img width="1847" height="866" alt="image" src="https://github.com/user-attachments/assets/c6fee330-9c6e-4c66-95d4-27128a27728c" /> 

---

#### Checklist  
- Added CTA buttons (“Explore Programs” and “Get started).  
- Fixed navbar alignment and added sticky + hover effects.  
- Improved hero section layout, visuals, and typography.  
- Added responsive design for mobile and tablet.  
- Updated footer and other sections for design consistency.  
- Verified no overlapping or misalignment on any screen size.  

---

Sandraa16012007 committed:
Fixed hero section, navbar and landing page sections

VenusCh001 commented:
A video or deployed link kindly share @Sandraa16012007

Sandraa16012007 commented:
> A video or deployed link kindly share @Sandraa16012007

Deployed link: https://sandraa16012007.github.io/thementorshipclub/

VenusCh001 commented:
Also what about mobile and tablet responsiveness @Sandraa16012007

Sandraa16012007 commented:
> Also what about mobile and tablet responsiveness @Sandraa16012007

It is responsive 
<img width="846" height="903" alt="image" src="https://github.com/user-attachments/assets/e418b1cb-7f26-43c0-9afb-3d1d76c509d4" />

VenusCh001 commented:
@Sandraa16012007 the hamburger is not working in the mobile menu and please maintain the orange colour coding 
Also reduce the sizes of the contact and about us boxes they are very huge

Sandraa16012007 committed:
Apply requested changes to hero section, hamburger menu and cards

Sandraa16012007 commented:
> @Sandraa16012007 the hamburger is not working in the mobile menu and please maintain the orange colour coding Also reduce the sizes of the contact and about us boxes they are very huge

I have made the necessary changes, the hamburger menu works and the color coding is the same as the original website. Teh sizes of the boxes has also been reduced.

VenusCh001 commented:
#ff5722
Please do add this colour for the main heading and subheadings but currently it is linear gradient .
@Sandraa16012007

Sandraa16012007 committed:
Changed colors and styles of headings and logo

Sandraa16012007 commented:
> #ff5722 Please do add this colour for the main heading and subheadings but currently it is linear gradient . @Sandraa16012007

Done! Is it alright now?

VenusCh001 commented:
<img width="1919" height="909" alt="Screenshot 2025-10-25 173831" src="https://github.com/user-attachments/assets/82611aab-8ccb-4a35-8f7d-02c44e2c15e8" />

Hey @Sandraa16012007 its good work till now !!

But i would like to know why the transform animations are added like you can inspect it in the window and check out the area near the hamburger and the element is seemed to be rotating

<img width="1919" height="914" alt="image" src="https://github.com/user-attachments/assets/98a93a51-3f7e-4761-a12e-b9bc4cc6375a" />

Sandraa16012007 committed:
Removed gradient rotating animation

Sandraa16012007 commented:
> <img alt="Screenshot 2025-10-25 173831" width="1919" height="909" src="https://private-user-images.githubusercontent.com/122210300/505629089-82611aab-8ccb-4a35-8f7d-02c44e2c15e8.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjEzOTc4NzIsIm5iZiI6MTc2MTM5NzU3MiwicGF0aCI6Ii8xMjIyMTAzMDAvNTA1NjI5MDg5LTgyNjExYWFiLThjY2ItNGEzNS04ZjdkLTAyYzQ0ZTJjMTVlOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyNVQxMzA2MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNTFlMGJiMzljNjZlYTM1Y2U2YjYzZjVlMTJlMmNhNDQ3OWI1YzI5Nzc0ZDI0MTc2MjAwMGJlZWMwMjVkYjczJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Up9_ZkTWaJpXuRqXDzWpsQMRjddMujvaEpupDjanvgA">
> Hey @Sandraa16012007 its good work till now !!
> 
> But i would like to know why the transform animations are added like you can inspect it in the window and check out the area near the hamburger and the element is seemed to be rotating
> 
> <img alt="image" width="1919" height="914" src="https://private-user-images.githubusercontent.com/122210300/505629327-98a93a51-3f7e-4761-a12e-b9bc4cc6375a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjEzOTc4NzIsIm5iZiI6MTc2MTM5NzU3MiwicGF0aCI6Ii8xMjIyMTAzMDAvNTA1NjI5MzI3LTk4YTkzYTUxLTNmN2UtNDc2MS1hMTJlLWI5YmM0Y2M2Mzc1YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyNVQxMzA2MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMDY1NjBmMjZhZTlkYzg2NTE4NGZiYzllODVmZjk2NjBjZmRmOGExYjhjOWQ0NDdmYTc2MGUyMjY4MzU1NDlhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-YFd4wbwsTITFOkFK96jVYxd4mst-0XEdv8l9D7z55s">

It was just a rotating gradient that was rotating which I added just for aesthetic purposes. I have removed it now!
